js/main.js

```diff
@@ -212,16 +213,20 @@ Site.Layout = {
   },
 
   convertScroll: function() {
-    if ($('body').css('overflow-x') == 'scroll') {
+    var _this = this;
+
+    if ($('body').css('overflow-x') == 'scroll' && _this.windowWidth >= 720) {
```

Comment:
should the `720` value here come from a variable on `Site` as has happened on some sites that need a mobile threshold?

Reply:
sure go for it